Package.swift

```diff
@@ -134,9 +141,19 @@ let package = Package(
             name: "SmithyHTTPAPI",
             dependencies: ["Smithy"]
         ),
+        .target(
+            name: "SmithyHTTPAuth",
+            dependencies: [
+                "Smithy",
+                "SmithyHTTPAuthAPI",
+                "SmithyIdentity",
+                "SmithyIdentityAPI",
+                .product(name: "AwsCommonRuntimeKit", package: "aws-crt-swift")
+            ]
+        ),
         .target(
             name: "SmithyHTTPAuthAPI",
-            dependencies: ["Smithy", "SmithyHTTPAPI"]
+            dependencies: ["Smithy", "SmithyHTTPAPI", "SmithyIdentityAPI", "SmithyChecksumsAPI"]
```

Comment:
SmithyChecksumsAPI shouldnt be a dependency here

Reply:
Removed as pointed out